### ISSUE TITLE
Interpreter should allow statements to be split into multiple lines

### ISSUE THREAD

**Issue by queiroz** [edge-js/edge#34]
e.g.

`{{ elIf('<span class="helper-text" data-error="$self"></span>', getErrorFor('email'), hasErrorFor('email')) }}`

**Comment by thetutlage** [comment]
Currently it does not. You can use the `@mustache` tag for that.

```
@mustache
	elIf(
		'<span class="helper-text" data-error="$self"></span>',
		getErrorFor('email'),
		hasErrorFor('email')
	)
@endmustache
```

**Comment by thetutlage** [comment]
Fixed in `v3.0.0`. It has breaking changes and will be out by end of the week
